source/Octopus.Tentacle.Client.Tests/ScriptOrchestratorFactoryFixture.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NUnit.Framework;
+using Octopus.Tentacle.Client.Scripts;
+using Octopus.Tentacle.Client.Tests.Builders;
+using Octopus.Tentacle.Contracts.ScriptServiceV2;
+
+namespace Octopus.Tentacle.Client.Tests
+{
+    [TestFixture]
+    public class ScriptOrchestratorFactoryFixture
+    {
+        readonly CancellationTokenSource cancellationTokenSource = new(TimeSpan.FromSeconds(42));
```

Comment:
I applied this rather generous timeout to the tests. The substitutes used in these tests will mean there are no network requests, so I'm not expecting the cancellation token to ever be used.

Please let me know if there is a preferred approach to take.

Reply:
Drop this comment in the code.

Also `42` ha nice